src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/ByteArrayToStorageQueueMessageConverter.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using Microsoft.Azure.WebJobs.Host.Converters;
-using Microsoft.Azure.WebJobs.Host.Storage.Queue;
-
-namespace Microsoft.Azure.WebJobs.Host.Queues.Bindings
-{
-    internal class ByteArrayToStorageQueueMessageConverter : IConverter<byte[], IStorageQueueMessage>
-    {
-        private readonly IStorageQueue _queue;
-
-        public ByteArrayToStorageQueueMessageConverter(IStorageQueue queue)
-        {
-            if (queue == null)
-            {
-                throw new ArgumentNullException("queue");
-            }
-
-            _queue = queue;
-        }
-
-        public IStorageQueueMessage Convert(byte[] input)
-        {
-            if (input == null)
-            {
-                throw new InvalidOperationException("A queue message cannot contain a null byte array instance.");
```

Comment:
In this specific case, this is an inconsistency and we should allow null and map it to "don't queue an object". 
I believe the errors are handled in both cases. In general, the declarative system lets us give rich errors. 

---
In reply to: [62360641](https://github.com/Azure/azure-webjobs-sdk/pull/711#discussion_r62360641) [](ancestors = 62360641)